src/main/kotlin/com/nextstep/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,21 @@
+package com.nextstep.racingcar.domain
+
+import com.nextstep.racingcar.domain.rules.MoveRule
+
+class Cars(cars: List<Car>) {
+    private val cars: List<Car>
+    init {
+        this.cars = cars
+    }
```

Comment:
이 부분은,
```suggestion
class Cars(private val cars: List<Car>) {

```
위와 크게 다르지 않습니다 :) 어떤 점을 의도하신 건지 궁금해지네요!!

---

외부에서 MutableList의 구현체를 넘겨주고 Cars를 생성한다면, 외부에서 참조중인 MutableList에 add, remove 등을 했을 때 Cars의 내용물이 변할 수 있습니다.
이 점을 생각하셨다면, init 부분에서 생성자 파라미터로 받은 cars를 한 번 복사해서 할당해준다면 위 이슈가 해결됩니다 :)

cars.toList() 를 하면 간단하게 List 객체를 얕게 복사할 수 있습니다 :)

Reply:
앗....ㅎ 의도한건 아니고 리팩토링하다가 졸려서 놓친 것 같아요 ㅎㅎ..
다만 코멘트 주신 부분은 생각해볼 부분이네요! 깊은 복사(?)로 넘겨주는 게 더 좋을 것 같네요~~!